client-app/src/container/project/new/ProjectNewApp.js

```diff
@@ -326,7 +374,30 @@ class ProjectNewApp extends Component {
             participationsInteressed,
         } = this.state.project;
 
-        const projectType = this.props.projectTypes.find(projectType => projectType.id == projectTypeId);
+        const projectType = this.props.projectTypesActive.find(projectType => projectType.id == projectTypeId);
+
+        // Benodigd aantal deelnemers: is opgesteld vermogen delen door Deelnemers per kWp van soort project
+        //  zonne-energieprojecten: Minimaal één deelnemer per 5 kWp vermogen;
+        //  windprojecten: minimaal één deelnemer per 2 kWp vermogen;
+        //	waterkracht: mimimaal één deelnemer per 1 kWp vermogen;
+        let requiredParticipations = 0;
+
+        switch (baseProjectCodeRef) {
```

Comment:
Beter om hiervoor een helper functie aan te maken. Zag deze switch namelijk ook op andere plekken.

Reply:
Cleanup code.
Created code RequiredParticipationsHelper to determine required participations. Same code (switch) was used in 3 different code files.